src/Tasks/FindReferenceAssembliesForReferences.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Ignore this file, totally forgot to remove it when I was testing something with item metadata